connect_proto.gemspec

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |s|
   s.name        = "primary_connect_proto"
-  s.version     = "0.10.1"
+  s.version     = "0.10.2"
```

Comment:
I guess technically this was a breaking change, and should have changed to `1.0.0` per [Semver](https://semver.org/). 

> Given a version number `MAJOR.MINOR.PATCH`, increment the:
> 
>     `MAJOR` version when you make incompatible API changes,
>     `MINOR` version when you add functionality in a backwards compatible manner, and
>     `PATCH` version when you make backwards compatible bug fixes.
> 
> Additional labels for pre-release and build metadata are available as extensions to the `MAJOR.MINOR.PATCH` format.

Ah well, not a big deal at this time, but something to keep in mind in the future